tododb/redis.go

```diff
@@ -20,7 +20,14 @@ type RedisDB struct {
 	appVersion     string
 }
 
-const redisKey = "todo"
+type RedisRole string
+
+const (
+	redisKey    string    = "todo"
+	okString    string    = "ok"
+	REDIS_SLAVE RedisRole = "redis-slave"
+	REDIS_MASTER RedisRole = "redis-master"
+)
```

Comment:
I would like to prevent adding variables/constants that are not needed you can use `redisDB.master` and `redisDB.slave` for this purpose with less code modifications see here: https://gist.github.com/johscheuer/7dbc435dc7591840508cb40559efea6e